File: WYNK.Data/Model/ViewModel/TransactionTypeViewM.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WYNK.Data.Model.ViewModel
{
  public  class TransactionTypeViewM
    {
        public TransactionType TransactionType { get; set; }
        public ICollection<getContraDet> getContraDet { get; set; }
    }

    public class getContraDet
    {
        public int TransactionID { get; set; }
        public string Description { get; set; }

    }



    }

File: WYNKWeb/src/app/Models/ViewModels/TransactionTypeViewM.ts
import { TransactionType } from '../transaction-type.model';


export class TransactionTypeViewM {


  TransactionType: TransactionType = new TransactionType();

  getContraDet: Array<getContraDet> = [];

}


export class getContraDet {


  TransactionID: number;
  Description: string;
}
